### PR TITLE
JS: refactor UriLibraries.qll models to use `DataFlow::moduleMember`

### PR DESCRIPTION
**Pull request by esben-semmle**
As discussed.

I do not think a change note is required for each library.

### CODE REVIEW

File: javascript/ql/src/semmle/javascript/frameworks/UriLibraries.qll [hunk 1]
```diff
@@ -94,10 +94,10 @@ module urijs {
 module uridashjs {
 
   /**
-   * Gets a data flow source node for the uridashjs library.
+   * Gets a data flow source node for member `name` of the uridashjs library.
    */
-  DataFlow::SourceNode uridashjs() {
-    result = DataFlow::moduleImport("uri-js")
+  DataFlow::SourceNode uridashjsMember(string name) {
```
Comment (xiemaisi) [inline javascript/ql/src/semmle/javascript/frameworks/UriLibraries.qll]: Can we make this predicate (and its brethren below) private?

**Comment by asger-semmle** [comment]
LGTM. Did you check for any new results?

File: javascript/ql/src/semmle/javascript/frameworks/UriLibraries.qll (hunk 1 above)
Comment (esben-semmle) [reply javascript/ql/src/semmle/javascript/frameworks/UriLibraries.qll]: I can, but that will break all of the library-specific sanity tests.

Comment (xiemaisi) [reply javascript/ql/src/semmle/javascript/frameworks/UriLibraries.qll]: Hm, OK. Never mind, then.

**Comment by esben-semmle** [comment]
> LGTM. Did you check for any new results?

No, the workers are too busy currently. I will try to do a local evaluation tonight.

**Comment by esben-semmle** [comment]
This PR adds no new results for our benchmark projects.

**Comment by xiemaisi** [comment]
This is a backwards-incompatible change since `UriLibraries.qll` was in 1.17 and `urijs::urijs()` was a public predicate. So we either need to add a change note or reintroduce the old predicates under deprecation; I'd prefer the latter.

@esben-semmle, in future let's not expose predicates like these unless we absolutely have to. In this case, I think we could have avoided this by writing one set of tests for the whole library, which in turn would have avoided the backwards compatibility issues.

**Comment by esben-semmle** [comment]
Deprecation markers added.

**Comment by xiemaisi** [comment]
Thanks; could you squash the fixup, please?

**Comment by esben-semmle** [comment]
Done

**Pull request merged**